### PR TITLE
Add solid:InsertDeletePatch

### LINKED ISSUE #66: Add `solid:InsertDeletePatch`
The solid protocol refers to this type but it is not defined in the vocab. See https://solidproject.org/ED/protocol#server-patch-n3-type

### PR DESCRIPTION
**Pull request by csarven**
Resolves #66

### CODE REVIEW

**Review by kjetilk — APPROVED** [review]
:+1:

File: solid-terms.ttl [hunk 1]
```diff
@@ -39,6 +39,15 @@ solid:Inbox
...
     rdfs:label "Inbox"@en .
 
+solid:InsertDeletePatch
+    a rdfs:Class ;
+    dc:issued "2022-05-03"^^xsd:date ;
+    rdfs:subClassOf solid:Patch ;
+    rdfs:comment "A class of patch expressing insertions, deletions, and conditional modifications to a resource that has an RDF-based representation."@en ;
+    rdfs:seeAlso <https://solidproject.org/TR/2021/protocol-20211217#server-patch-n3-type> ;
+    rdfs:isDefinedBy <http://www.w3.org/ns/solid/terms#> ;
+    rdfs:label "InsertDeletePatch"@en .
```
Comment (RubenVerborgh) [inline solid-terms.ttl:49]:
```suggestion
    rdfs:label "Insert/delete patch"@en .
```

**Review by RubenVerborgh — APPROVED** [review]

**Comment by damooo** [comment]
Hello @csarven , It seems not merged, and vocab file available at <http://www.w3.org/ns/solid/terms> doesn't contain this term either. For libraries that generates terms from their distribution ontology files, it is not possible to include and use it otherwise.

**Comment by csarven** [comment]
@timbl , we believe this is ready to go. Please have a last look, .. merge, and then publish `solid-terms.ttl` in `main` branch's HEAD.

**Pull request merged**